doofinder-for-woocommerce/includes/class-setup-wizard.php

```diff
@@ -1633,8 +1633,9 @@ private function set_search_engines($search_engines)
 		$currency = get_woocommerce_currency();
 		foreach ($search_engines as $language => $search_engine) {
 			$currency_key = strtoupper($currency);
-			$language_key = $language;
-			$is_primary_language = strtolower($this->language->get_base_language()) == strtolower($language);
+			//format language to en_US instead of en-US format			
+			$language_key = Helpers::format_locale_to_underscore($language);
+			$is_primary_language = strtolower($this->language->get_base_language()) == strtolower($language_key);

```

Comment:
As a good practice, we should get used to compare two values using the triple equal operator '==='